README.md

```diff
@@ -27,8 +20,53 @@ Fusebot was created for developers by developers at [fusebit.io](https://fusebit
 ## Installing Fusebot
 1. [Download Fusebot on Discord or Slack](https://fusebot.io?utm_source=github.com&utm_medium=referral&utm_campaign=fusebot-readme&utm_content=install)
 2. Run ```/fusebot coin DOGE``` to test command
-3. Run ```/fusebot edit coin``` to review code
-4. Run ```/fusebot help``` get started on your first command
+3. Run ```/fusebot edit coin``` and click the edit link to review how the code works
+
+## Code your first command
+
+Now that you have Fusebot installed and you tested the ```/fusebot coin``` command, let's get started on building your first custom command. Let's build a command search reddit. It is best to do development on a desktop browser as you will have more room to work with tabs and copy/paste code.
+
+1. Type ```/fusebot edit reddit``` and hit enter
+
+![image](https://user-images.githubusercontent.com/751491/126013119-033caee3-d6bd-4dcf-be6b-8025fe655261.png)
+
+2. Click the "click here" link
+
+![image](https://user-images.githubusercontent.com/751491/126013074-30368c7a-e5e2-4c86-a83d-979c2453c906.png)
+
+Your browser will open a window like this:
+
+![image](https://user-images.githubusercontent.com/751491/126013397-720bfe76-1909-4f9e-8a3a-2ccd554e835f.png)
+
+3. Open this [command.js file](https://github.com/fusebit/fusebot/blob/main/samples/reddit/command.js) in another tab
+
+![image](https://user-images.githubusercontent.com/751491/126013219-d0f5169b-ae65-4a63-b8d0-afda9dc04369.png)
```

Comment:
The mechanics of this is perhaps more complicated than it needs to be. Maybe we can make the code a bit more compact and just present it inline so that folks can copy & paste without opening new tabs or leaving the single track of the readme?

Reply:
yeah, it is clunky, but let's try it and see if anything changes before we make it even easier.